restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/fcm/FcmConfigurationClient.java

```diff
@@ -116,6 +117,9 @@ public String getAccountSid(){
 
         } catch (Exception ex){
             RCLogger.e(TAG, ex.toString());
+            if (ex instanceof UnknownHostException){
```

Comment:
Are here any other additional exceptions we might be interested in catching and returning an error code for apart from UnknownHostException? Like stuff that can go wrong and for which we want to be ready for? For example what happens if the request times out?

Reply:
I think the timeout will not rise an exception; It will return the http error code. The only think I can think of is MalformedURLException but we dont have that case with push. I think we are good with this one. We can talk more on daily.

Reply:
Ah right, well in that case how about adding handling and return codes for responseCode being:
1. HTTP_UNAUTHORIZED & HTTP_FORBIDDEN (same error code for RCDevice, as it should be more or less the same thing)
2. HTTP_NOT_FOUND
3. HTTP_INTERNAL_ERROR
4. HTTP_CLIENT_TIMEOUT

Since these are the most common to happen, and right now if they do it will be hard to figure our what is going on. Also, on (1) I think it also makes sense to introduce a new Integration TC using correct username but wrong password and assert that we got 'unauthorized'. For (2), (3), (4) I don't think we can add TCs as we can't really control them. 

Wdyt?